dao-generator/src/main/java/com/exactpro/cradle/cassandra/dao/loader/impl/DefaultDaoMapperLoader.java

```diff
@@ -9,10 +25,13 @@
 
 import com.datastax.oss.driver.api.mapper.annotations.Mapper;
 import com.exactpro.cradle.cassandra.dao.loader.DaoMapperLoader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class DefaultDaoMapperLoader implements DaoMapperLoader {
 
     private static final String DEFAULT_PATH_FOR_RESOURCE = "META-INF/dao";
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultDaoMapperLoader.class.getName());
```

Comment:
Use LOGGER wherever you need to print something in log. Don't use `e.printStartTrace()` like you did in line 56.